ManageRDSessions/ManageRDSessions.psm1

```diff
@@ -1,14 +1,114 @@
 #Requires -RunAsAdministrator
+
 function Get-sbRDSession {
```

Comment:
Make sure the CBH is aligned, and I'm not sure you should include the output within your examples (see my second example below:

```function Get-sbRDSession {
    <#
    .SYNOPSIS
        Get current sessions in a Remote Desktop Services deployment.
    .DESCRIPTION
        This cmdlet will return all Sessions in the current Remote Desktop Services deployment. Narrow results by user,
        or by a combination of minimum number of Idle Session minutes, Session State (Active, Disconnected, etc.) and choose
        whether to include yourself (the Admin user).
    .PARAMETER SessionState
        Returns only Remote Desktop sessions matching the specified State.
        Accepts: Active, Any, Connected, Disconnected or Idle.
    .PARAMETER IncludeSelf
        When enabled, this parameter will include the current (console) user within any matching search results. The
        current user is omitted by default as we don't normally want to send messages to/disconnect/log off ourselves.
    .PARAMETER MinimumIdleMins
        Specifies the minimum number of minutes for which each returned Remote Desktop should have been idle.
    .PARAMETER UserName
        Returns only Remote Desktop sessions matching the specified user name(s).
    .EXAMPLE
        Get-sbRDSession
        HostServer       : rds01.lab.milliondollar.me.uk
        UserName         : steve.baker
        UnifiedSessionID : 3
        SessionState     : STATE_ACTIVE
        IdleTime         : 0

        Getting all Remote Desktop sessions, regardless of status.
    .EXAMPLE
        Get-sbRDSession

        Getting all Remote Desktop sessions, regardless of status.        
    #>

    [OutputType('Custom.SB.RDSession')]
    ...
```

Reply:
Weirdly, the indentation locally matches the examples you've given below. I've indented the ending #> on each help block and re-pushed, let's see how it looks on GH